src/addon.js

```diff
@@ -135,6 +135,16 @@ class Addon extends EventEmitter {
     });
   }
 
+  openPremiumModal() {
```

Comment:
I think this should be `upgradePremium` as discussed with @siboulet https://wealthica.slack.com/archives/CDEL7QXHP/p1601949447033000?thread_ts=1599149392.014500&cid=CDEL7QXHP

Reply:
👍 thanks @codynguyen

Reply:
I was out of inspiration when I came up with `upgradePremium`, `openPremiumModal` was good too :) Anyway let's stick with upgradePremium.